testplugins/src/main/java/org/spongepowered/test/IndividualPluginReloadTest.java

```diff
@@ -0,0 +1,22 @@
+package org.spongepowered.test;
+
+import com.google.inject.Inject;
+import org.slf4j.Logger;
+import org.spongepowered.api.event.Listener;
+import org.spongepowered.api.event.game.GameReloadEvent;
+import org.spongepowered.api.plugin.Plugin;
+
+@Plugin(id="individual-plugin-reload-test",
+        name="Individual Plugin Reload Test",
+        description = "Tests sending GameReloadEvent to specific plugins.",
+        version = "1.0.0")
+public class IndividualPluginReloadTest {
+
+    @Inject private Logger logger;
+
+    @Listener
+    public void onReload(GameReloadEvent event) {
+        this.logger.info("GameReloadEvent: " + event);
```

Comment:
Being **incredibly** picky, aka I don't expect you to do this.

2 test plugins would be a ton better.

/sponge reload 

will trigger this message, and unless any of the other test plugins have messages on reload, or startup (I didn't check) you would never know it was reloading everything.

2 would let you pick between them.

But I'm sure it's fine as-is since you could issue the reload command to a different testplugin anyway.